cloud/azure/app-gateway/variables.tf

```
@@ -0,0 +1,655 @@
+  type        = string
+  default     = ".sum(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+}
+
+variable "http_5xx_errors_transformation_function" {
+  description = "Transformation function for http_5xx_errors detector (mean, min, max)"
+  type        = string
+  default     = "max"
+}
+
+variable "http_5xx_errors_transformation_window" {
+  description = "Transformation window for http_5xx_errors detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "5m"
+}
+
+variable "http_5xx_errors_threshold_critical" {
+  description = "Critical threshold for http_5xx_errors detector"
+  type        = number
+  default     = 95
+}
+
+variable "http_5xx_errors_threshold_warning" {
+  description = "Warning threshold for http_5xx_errors detector"
+  type        = number
+  default     = 80
+}
+
+variable "http_5xx_errors_aperiodic_duration" {
+  description = "Duration for the http_5xx_errors block"
+  type        = string
+  default     = "10m"
+}
+
+variable "http_5xx_errors_aperiodic_percentage" {
+  description = "Percentage for the http_5xx_errors block"
+  type        = number
+  default     = 0.9
+}
+
+# Backend_http_4xx_errors detectors
+
+variable "backend_http_4xx_errors_disabled" {
+  description = "Disable all alerting rules for backend_http_4xx_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "backend_http_4xx_errors_disabled_critical" {
+  description = "Disable critical alerting rule for backend_http_4xx_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "backend_http_4xx_errors_disabled_warning" {
+  description = "Disable warning alerting rule for backend_http_4xx_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "backend_http_4xx_errors_notifications" {
+  description = "Notification recipients list for every alerting rules of backend_http_4xx_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "backend_http_4xx_errors_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of backend_http_4xx_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "backend_http_4xx_errors_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of backend_http_4xx_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "backend_http_4xx_errors_aggregation_function" {
+  description = "Aggregation function and group by for backend_http_4xx_errors detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region', 'backendhttpsetting', 'backendpool', 'backendserver'])"
+}
+
+variable "backend_http_4xx_errors_transformation_function" {
+  description = "Transformation function for backend_http_4xx_errors detector (mean, min, max)"
+  type        = string
+  default     = "max"
```

```suggestion
  default     = "min"
```
This transformation was inverted in previous implementation.